vendor/clink.lua

```diff
@@ -666,10 +677,15 @@ for _,lua_module in ipairs(clink.find_files(completions_dir..'*.lua')) do
     end
 end
 
-if clink.get_env('CMDER_USER_CONFIG') then
-    local cmder_config_dir = clink.get_env('CMDER_ROOT')..'/config/'
-    for _,lua_module in ipairs(clink.find_files(cmder_config_dir..'*.lua')) do
-        local filename = cmder_config_dir..lua_module
+---
+-- If Cmder is launched with '/c [folderPath]', indicating Cmder is installed globally and
+-- each user has a private '[folderPath]\config' folder, Clink won't know about the global
+-- 'CMDER_CONFIG_PATH` dir, so we need to load scripts from there before Clink loads lua 
+-- scripts from the profile directory given to it when it was injected.
+---
+if not (isempty(CMDER_USER_CONFIG_PATH)) then
```

Comment:
I would recommend to add the comment, but don't change the code to use global Lua variables.  Using environment variables here is good.

I was recommending to add another environment variable in `init.bat` such as `set CMDER_DEFAULT_CONFIG_DIR=%CMDER_ROOT%\config`, and then use `%CMDER_DEFAULT_CONFIG_DIR%` in all the places instead of `%CMDER_ROOT%\config`.